CONTRIBUTING.md

```diff
@@ -41,9 +41,12 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. 
 
-1. Fork the repo and create your branch from `master`.
+Netlify CMS uses the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) + [Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows#feature-branch-workflow). Additionally, PR's should [rebase on rather than merge](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) master to stay current. 
+
+1. Fork the repo
+2. create a feature branch from `master`. If you're working on an issue, prefix your branch name with {{issue_number}}_description, otherwise use xx_description.
 2. If you've added code that should be tested, add tests.
```

Comment:
I'm wondering if we should say something at the end of this list like: "If you need any help following one of these steps, please note that in your PR, and we will be glad to help." @erquhart?

Reply:
@tech4him1 I like that idea; I don’t want this to stop anybody from contributing due to Git familiarity.